### DISCUSSION TITLE
How can I tell what distribution an LXC is based on?

### DISCUSSION THREAD

**Discussion by johnsturgeon** [tteck/Proxmox#3059] [General] [answered]
For example, I like `ubuntu`.   If I install the `sonarr` lxc, what distribution is it using?  How can I tell?

**Comment by tteck** [comment, marked as answer]
Default settings
https://github.com/tteck/Proxmox/blob/ea135820e36f70e49920cf6beaad5f6192381019/ct/sonarr.sh#L25-L26

**Reply by johnsturgeon** [reply]
🤦

**Reply by techie2000** [reply]
@johnsturgeon if you don't want to look at the sourcecode, you can do an advanced install and it will advise what the expected/required flavour and version is required in order to continue. Alternatively, once installed, go to the command line inside the container and type `cat /etc/os-release`
